node/src/types/status_feed.rs

```diff
@@ -185,28 +194,35 @@ pub struct GetStatusResult {
     pub round_length: Option<TimeDiff>,
     /// Information about the next scheduled upgrade.
     pub next_upgrade: Option<NextUpgrade>,
-    /// The compiled node version.
-    pub build_version: String,
     /// Time that passed since the node has started.
     pub uptime: TimeDiff,
-    /// The current state of node.
-    pub node_state: NodeState,
+    /// The current state of node reactor.
+    pub reactor_state: ReactorState,
+    /// Timestamp of the last recorded progress in the reactor.
+    pub last_progress: Timestamp,
+    /// The available block range in storage.
+    pub available_block_range: AvailableBlockRange,
+    /// The status of the block synchronizer builders.
+    pub block_sync: Vec<BlockSyncStatus>,
 }
 
 impl GetStatusResult {
     #[allow(deprecated)]
     pub(crate) fn new(status_feed: StatusFeed, api_version: ProtocolVersion) -> Self {
         GetStatusResult {
+            peers: PeersMap::from(status_feed.peers),
             api_version,
             chainspec_name: status_feed.chainspec_info.name,
-            starting_state_root_hash: Digest::from([0u8; 32]),
-            peers: PeersMap::from(status_feed.peers),
+            starting_state_root_hash: status_feed.starting_state_root_hash,
             last_added_block_info: status_feed.last_added_block.map(Into::into),
             our_public_signing_key: status_feed.our_public_signing_key,
             round_length: status_feed.round_length,
             next_upgrade: status_feed.chainspec_info.next_upgrade,
             uptime: status_feed.node_uptime.into(),
-            node_state: status_feed.node_state,
+            reactor_state: status_feed.reactor_state,
```

Comment:
Changelog still mentions `node_state`, but possibly we gonna have a separate ticket to clean-up the changelog. So I guess we can leave it be for now.